src/System.Diagnostics.FileVersionInfo/tests/NativeConsoleApp/Configurations.props

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I thought you were asking why we can't build it every time and not check in the binaries. Things make sense now.